lib/_io.scm

```diff
@@ -8343,6 +8343,167 @@
         (macro-dynamic-bind output-port port thunk)))))
 
 ;;;----------------------------------------------------------------------------
+(define-prim (##open-raw-device
+              direction
+              name
+              fd)
```

Comment:
Actually it's a convenient function, so that the user doesn't have to rewrite the get setting by hand all the time. I will rename the reference to `fd` to `device`